### PR TITLE
OfferFit partner docs

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_hidden/wip_partnerships/offerfit.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+You can leverage the OfferFit native activation integration with Braze to orchestrate and schedule 1:1 personalized recommendations for your target audience.
+
+## Customization
+
+In addition to orchestrating activation events in Braze, OfferFit provides data integration capabilities that allow you to retrieve customer profile (non-PII) and engagement data from Braze through the available API endpoints.
+
+## Using this integration
+
+After OfferFit is configured, the automated experimentation platform will automatically send 1:1 personalized activation events to Braze for each user in your target audience. These activation events will be triggered through the Braze campaigns or Canvases you configured in [step 2](#step-2-configure-an-api-triggered-braze-campaign-or-canvas-and-create-campaign-assets-eg-html-templates-images).
```
Comment (bre-fitzgerald) [inline _docs/_hidden/wip_partnerships/offerfit.md:116]: That's a pretty long anchor link, I'd recommend adding a custom one to shorten it.

```suggestion
After OfferFit is configured, the automated experimentation platform will automatically send 1:1 personalized activation events to Braze for each user in your target audience. These activation events will be triggered through the Braze campaigns or Canvases you configured in [step 2](#step-2).
```

File: _docs/_hidden/wip_partnerships/offerfit.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3 .reset-td-br-4}
+
+
+## Integration
+
+### Step 1: Define target audience in Braze
+
+Define your target audience by creating at least one segment in Braze. This segment will be used to send your campaign or Canvas to the right users.
+
+### Step 2: Configure an API-triggered Braze campaign or Canvas and create campaign assets (e.g., HTML templates, images)
```
Comment (bre-fitzgerald) [inline _docs/_hidden/wip_partnerships/offerfit.md:98]: Adds custom anchor link

```suggestion
### Step 2: Configure an API-triggered Braze campaign or Canvas and create campaign assets (e.g., HTML templates, images) {#step-2}
```

**Comment by rachel-feinberg** [comment]
Original PR and draft are located [here](https://github.com/braze-inc/braze-docs/pull/5966).

File: _docs/_hidden/wip_partnerships/offerfit.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+page_type: partner
+search_tag: OfferFit
+hidden: true
+layout: dev_guide
+---
+
+
+# OfferFit
+
+[OfferFit](https://www.offerfit.ai/) is a self-learning platform that automates the experimentation process that is traditionally done through A/B testing. This allows you to automatically discover the right message, channel, and timing for every customer based on your customer data. You can optimize your campaigns to existing identified customers, with business goals such as cross-sell, upsell, repurchase, retention, renewal, referral, and winback.
```
Comment (bre-fitzgerald) [inline _docs/_hidden/wip_partnerships/offerfit.md:20]: Formatting adjustment per partner docs template

```suggestion
> [OfferFit](https://www.offerfit.ai/) is a self-learning platform that automates the experimentation process that is traditionally done through A/B testing. 

The OfferFit and Braze integration allows you to automatically discover the right message, channel, and timing for every customer based on your customer data. You can optimize your campaigns to existing identified customers, with business goals such as cross-sell, upsell, repurchase, retention, renewal, referral, and winback.
```

File: _docs/_hidden/wip_partnerships/offerfit.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+| [POST /templates/email/create](https://www.braze.com/docs/api/endpoints/templates/email_templates/post_create_email_template/) | Create a new Braze HTML email template. |
+| [POST /templates/email/update](https://www.braze.com/docs/api/endpoints/templates/email_templates/post_update_email_template/) | Update an existing Braze HTML email template. |
+| [GET /templates/email/info](https://www.braze.com/docs/api/endpoints/templates/email_templates/get_see_email_template_information/) | Retrieve the details of a specific Braze HTML email template. |
+| [GET /templates/email/list](https://www.braze.com/docs/api/endpoints/templates/email_templates/get_list_email_templates/) | Retrieve the list of all the Braze HTML email templates configured in Braze and their `subject line` and `HTML content`. |
+{: .reset-td-br-1 .reset-td-br-2}
+
+
+{% alert note %}
+If you are using the [older navigation]({{site.baseurl}}/navigation), you can create an API key at **Developer Console** > **API Settings**.
+{% endalert %}
```
Comment (bre-fitzgerald) [inline _docs/_hidden/wip_partnerships/offerfit.md:60]: This note doesn't seem to be in the right place. I'd bump this up to directly after the prerequisites table, as it's referring to the third row item.

File: _docs/_hidden/wip_partnerships/offerfit.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+Define your target audience by creating at least one segment in Braze. This segment will be used to send your campaign or Canvas to the right users.
+
+### Step 2: Configure an API-triggered Braze campaign or Canvas and create campaign assets (e.g., HTML templates, images)
+
+First, create a campaign or Canvas in Braze. OfferFit will use this campaign or Canvas to send 1:1 personalized activation events to the right users from your defined audience. 
+
+Next, disable the Braze Control Group. This allows the OfferFit Control Group to be the only active one.
+
+Finally, depending on your dimensions, you can configure Liquid tags in your creative content to dynamically populate your campaign or Canvas with OfferFit recommendations. OfferFit will pass customer-specific content to the Liquid tags in your templates via the Braze API.
```
Comment (bre-fitzgerald) [inline _docs/_hidden/wip_partnerships/offerfit.md:104]: I think we can make this little bit into an actual numbered list for it to be more digestible. Also, I'd recommend linking out to the documentation on control groups so the reader knows we mean regular control groups and aren't talking about the Global Control Group. Control groups are not selected by default (it's a checkbox in the A/B testing panel) so it doesn't quite make sense to say disable or turn off.

```suggestion
1. Create a campaign or Canvas in Braze. OfferFit will use this campaign or Canvas to send 1:1 personalized activation events to the right users from your defined audience. 
2. Do not include a Braze [control group]({{site.baseurl}}/user_guide/engagement_tools/testing/multivariant_testing/create_multivariate_campaign#including-a-control-group) in your campaign or Canvas. This allows the OfferFit control group to be the only active one.
3. Depending on your dimensions, you can configure Liquid tags in your creative content to dynamically populate your campaign or Canvas with OfferFit recommendations. OfferFit will pass customer-specific content to the Liquid tags in your templates via the Braze API.
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Left some comments and suggestions for your review! Thanks for taking this on 💪

**Review by bre-fitzgerald — APPROVED** [review]
lgtm!

**Pull request merged**